### PR TITLE
[MRG] Add Min/Max Frequency and Add Visual Updates Ahead of 0.4

### PR DESCRIPTION
**Pull request by dylansdaniels**
Adding sections to Simulation tab, separating our the default values for visualization parameters that will be updated on 'run' 
Screenshot 2024-11-22 at 5 26 54 PM

Improvements to network connectivity tab, adjusting spacing of Vboxes, text indentation, adding css classes to more easily target sections, updating display of the receptor names
Screenshot 2024-11-22 at 5 27 14 PM

### CODE REVIEW

**Comment by dylansdaniels** [comment]
Note that the min/max spectral freq widgets are just placeholders here, they don't actually update the defaults like the Smoothing field does. But could add that to this or another PR

File: hnn_core/tests/test_gui.py [hunk 1]
```diff
@@ -1174,3 +1176,39 @@ def test_default_smoothing(setup_gui):
...
+    # the default simulation
+    gui.run_button.click()
+
+    gui_min = gui.fig_default_params['default_min_frequency']
+    viz_min = gui.viz_manager.fig_default_params['default_min_frequency']
+    gui_max = gui.fig_default_params['default_max_frequency']
+    viz_max = gui.viz_manager.fig_default_params['default_max_frequency']
+
+    assert gui_min == viz_min == 10
+    assert gui_max == viz_max == 100
```
Comment (asoplata) [inline hnn_core/tests/test_gui.py:1195]: Everything looks great, the only thing I would change is removing the current default frequency values here. I think it's enough to check that they're consistent. We may want to change the default frequency values in the future, but with this check, we would also have to remember to manually change these lines in the tests.

**Comment by dylansdaniels** [comment]
@asoplata made your change and similarly edited the initial equivalency check for test_default_smoothing!

**Review by asoplata — APPROVED** [review]
LGTM, tested the buttons in the GUI, everything seems to work

**Pull request merged**